src/certs/truststore.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDdTCCAl2gAwIBAgIUCOLpLY2W4NKdTrg2dwAgsu7pHZswDQYJKoZIhvcNAQEL
+BQAwYzELMAkGA1UEBhMCTkwxEzARBgNVBAgMCkdlbGRlcmxhbmQxETAPBgNVBAcM
+CE5pam1lZ2VuMRowGAYDVQQKDBFHZW1lZW50ZSBOaWptZWdlbjEQMA4GA1UEAwwH
+cGFydG5lcjAeFw0yNDEwMDExMzIxNDhaFw0yNTEwMDExMzIxNDhaMGMxCzAJBgNV
+BAYTAk5MMRMwEQYDVQQIDApHZWxkZXJsYW5kMREwDwYDVQQHDAhOaWptZWdlbjEa
+MBgGA1UECgwRR2VtZWVudGUgTmlqbWVnZW4xEDAOBgNVBAMMB3BhcnRuZXIwggEi
+MA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCHqjE1+p/k5tgmoTcwuIOFortB
+IjHh7q7CuARumHoqZrATu7ZASlJkWJcXKgkRuELR/Efa/mHyfZoRYaF53TG9/zcu
+izkkbY2dpdGGJms2R5oe8ERpwH4JTg0N9iek5C9pyWOL4KEwyt5owJhHP5TMb13y
+10KIInXRQKKGt3/omgFaKour9UE4rGdejWOcC14h309fqmQBkDf3b2ZiO9QFLSEJ
+6eXTkA+jgSDwLj1izELmWWXSevubewni2jCa1KWWvs9QDhbuuAtuVEVppO8o+FYP
+vAuFZEP8xJEmBpSfxjIWGgpnE38lFm6PX1TtrxiIpmwcicCnCgtyQMiUxIRnAgMB
+AAGjITAfMB0GA1UdDgQWBBT5I/Prq9cofqg9l3+9dzzLFoKm2TANBgkqhkiG9w0B
+AQsFAAOCAQEAduGDkOwMJbcyVQzd6uirnNV5q8yAXYVOO4aC+T/z8SAf1hpUQTl6
+FNl2vyTdmqoS8vfAz1b9JD93sB/0xvJLWYoOmiDqakfDEMQfQza/M7igofGYyfsQ
+8WV7WO5l3Wso/b1bnU2e9mJoiZh/QnvYc3WwWlWNIGZOAUwFOrrRmBRVU9LhiYV7
+cac+OKoK39ED7Vnspo4LTXt6AihXEWUQg1DfP7CGp1c3Ir/y71vjRhC1Kk2nMKD9
+TSLhttbxkOhvwOi6AFTJ20lQO21uvKBhhDxnXk0lttYtIHBQVImLqhSJey+kMGgD
+TcC+rfdXLdB2/8fXdcPktAI45/B3/Nojig==
+-----END CERTIFICATE-----
```

Comment:
Ik snap dat het een doel heeft, maar voelt onheilspellend om in Git te zien

Reply:
Is een self signed cert die niet wordt gebruikt